api/lib/domain/services/badge-criteria-service.js

```diff
@@ -27,6 +28,10 @@ function getMasteryPercentageForAllPartnerCompetences({ targetedKnowledgeElement
 }
 
 function verifyCriteriaFulfilment({ masteryPercentage, partnerCompetenceResults, badge }) {
+  if (badge.badgeCriteria.length === 0) {
```

Comment:
Il y a aussi cette solution, je sais pas ce qu'on préfère chez Pix : 
```suggestion
  if (!badge.badgeCriteria.length) {
```